server.js

```diff
@@ -118,18 +118,19 @@ app.get('/api/v1/comments', (request, response) => {
 });
 
 
-app.post('/api/v1/discussions', checkAuth, (request, response) => {
```

Comment:
I agree about potentially removing this, let's chat about it more as we move past MVP.